### PR TITLE
Fix linting errors in tasks table page

### PR DESCRIPTION
**Pull request by Calvinp**
Fixes the linting errors in the components of the Tasks table. (There were no linting errors in the view, action, or controller).

### CODE REVIEW

File: SingularityUI/app/components/tasks/Columns.jsx [hunk 1]
```diff
@@ -116,26 +116,26 @@ export const Memory = (
...
-  return <Column
-    label=""
-    id="actions"
-    key="actions"
-    className="actions-column"
-    cellData={
-      (rowData) => rowData
-    }
-    cellRender={(cellData) => {
+  return (
```
Comment (tpetr) [inline SingularityUI/app/components/tasks/Columns.jsx]: is the return here part of our style guide? i dont necessarily have an issue with it, but am curious if it's there for an explicit reason

**Comment by tpetr** [comment]
LGTM aside from question above

File: SingularityUI/app/components/tasks/Columns.jsx (hunk 1 above)
Comment (Calvinp) [inline SingularityUI/app/components/tasks/Columns.jsx]: It's not - it would be perfectly fine to get rid of it entirely.

**Comment by Calvinp** [comment]
I'm going to merge this since the above question has been answered.

**Pull request merged**